draft-ietf-cose-hash-envelope.md

```diff
@@ -87,9 +87,12 @@ TBD_1:
   : the hash algorithm used to produce the payload.
 
 TBD_2:
-  : the content type of the bytes that were hashed (preimage) to produce the payload, given as a content-format number ({{Section 12.3 of RFC7252}}) or as a media-type name optionally with parameters ({{Section 8.3 of RFC9110}}).
+  : the length of the bytes that were hashed (pre-image) to produce the payload.
```

Comment:
No.

Reply:
No, what? Not helpful

Reply:
@OR13 I get that the creator of the envelope knows the length, but how would a verify know how much content to fetch from a remote location ?

Reply:
This would make the hash over a partial of the original file...

In the case that the file was a large image, a hash over part of it, has little utility.

A clever adversary could then modify all the bytes after the length, without impacting the hash.

I don't support the proposed changes, and I need more information to understand how this is supposed to improve the security of the draft.